recipes/zyre/2.0.0/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class ZyreConan(ConanFile):
+    name = "zyre"
+    version = "2.0.0"
```

Comment:
```suggestion
```

This recipe can generic I think. I can't any specific condition here which is only for a specific version.

Reply:
Thanks for the hint but it seems to be necessary `ERROR: conanfile didn't specify version`.

Reply:
because you didn't add a config.yml to index it. I'll open a PR suggesting this change.